tools/codegen/core/gen_static_metadata.py

```diff
@@ -595,8 +594,25 @@ def f(i, p=p):
 print >> H, '  } named;'
 print >> H, '} grpc_metadata_batch_callouts;'
 print >> H
-print >> H, '#define GRPC_BATCH_INDEX_OF(slice) \\'
-print >> H, '  (GRPC_IS_STATIC_METADATA_STRING((slice)) ? static_cast<grpc_metadata_batch_callouts_index>(GPR_CLAMP(GRPC_STATIC_METADATA_INDEX((slice)), 0, static_cast<intptr_t>(GRPC_BATCH_CALLOUTS_COUNT))) : GRPC_BATCH_CALLOUTS_COUNT)'
+
+batch_idx_of_hdr = '#define GRPC_BATCH_INDEX_OF(slice) \\'
+static_slice = 'GRPC_IS_STATIC_METADATA_STRING((slice))'
+slice2sref = '(slice).refcount'
+static_sref_type = 'grpc_core::StaticSliceRefcount*'
+rcast = 'reinterpret_cast'
```

Comment:
please spell these out in places they happen. you can go above the 80-column limit here.

Reply:
I meant basically removing `scast` and `rcast`.

Reply:
Already fixed the names in response to @hcaseyal .

Regarding why I'm using these short variable names at all - for the really cumbersome stuff like the casts, the final join() statement that assembles things is a lot easier to parse without the awful C++ syntax gumming it up.